client/my-sites/customer-home/components/celebrate-launch-modal.jsx

```diff
@@ -20,53 +16,36 @@ function CelebrateLaunchModal( { setModalIsOpen, site, allDomains } ) {
 	const translate = useTranslate();
 	const isPaidPlan = ! site?.plan?.is_free;
 	const isBilledMonthly = site?.plan?.product_slug?.includes( 'monthly' );
+
 	const transformedDomains = allDomains.map( createSiteDomainObject );
-	const [ isInitiallyLoaded, setIsInitiallyLoaded ] = useState( false );
 	const [ clipboardCopied, setClipboardCopied ] = useState( false );
 	const clipboardButtonEl = useRef( null );
 	const hasCustomDomain = Boolean(
 		transformedDomains.find( ( domain ) => ! domain.isWPCOMDomain )
 	);
-	const sitePlanSlug = useSelector( ( state ) => getSitePlanSlug( state, site?.ID ) );
-	const purchases = useSelector( ( state ) => getSitePurchases( state, site?.ID ) );
-	const actualPlanPurchase = purchases.filter(
-		( purchase ) => purchase.productSlug === sitePlanSlug
-	);
-	const isA4ASite = isA4APurchase( actualPlanPurchase[ 0 ] );
-	const isLoading = useSelector( isFetchingSitePurchases );
 
 	useEffect( () => {
-		// Remove the celebrateLaunch URL param without reloading the page as soon as the modal loads
-		// Make sure the modal is shown only once
+		// remove the celebrateLaunch URL param without reloading the page as soon as the modal loads
+		// make sure the modal is shown only once
```

Comment:
This is just a minor thing: I think we could have left this fix in (similar to the fixes below where we are replacing backticks with singe quotes). 🙂